public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -384,7 +386,13 @@ export class GeomapPanel extends Component<Props, State> {
       <>
         <Global styles={this.globalCSS} />
         <div className={styles.wrap} onMouseLeave={this.clearTooltip}>
-          <div className={styles.map} ref={this.initMapRef}></div>
+          <section
```

Comment:
This probably doesn't need to be a landmark element, as the aria-description can't be applied to it.